incent-config/Dangerfile

```diff
@@ -156,7 +160,8 @@ end
 
 if rendered_programs_changed?
   rendered_programs_changed_message = <<~MESSAGE
-    Changes occurred while rendering.
+    Changes occurred while rendering in:
+    #{rendered_programs_changes}.
     Please generate files from templates before pushing by running:
     `incent-validate-program-set`.
```

Comment:
Might as well take the opportunity to remove or update this part (e.g. run validations in VS Code, not shell).